settings.gradle

```diff
@@ -1 +1,32 @@
+pluginManagement {
```

Comment:
`buildSrc` and the root project are actually two entirely separate projects. But `buildSrc` is a special folder that gradle knows to look for and utilize. Which is why I also have to do `./gradlew -p buildSrc clean` instead of `./gradlew :buildSrc:clean`